sdk/communication/azure-communication-callautomation/src/main/java/com/azure/communication/callautomation/implementation/models/CollectTonesResult.java

```diff
@@ -15,14 +15,14 @@ public final class CollectTonesResult {
      * The tones property.
      */
     @JsonProperty(value = "tones", access = JsonProperty.Access.WRITE_ONLY)
-    private List<DtmfTone> tones;
+    private List<DtmfToneInternal> tones;
```

Comment:
Question about this `CollectTonesResult`. Are you exposing this directly from the implementation folder? Or you want to make this internal as well with an external representation?

Reply:
CollectTonesResult is in public folder, which is consistent with .net sdk.

Reply:
For my own sake, how's this different than DtmfTone that existed within the same folder but was marked as Internal?

Reply:
The DtmfTone name is changed to DtmfToneInternal in generated models, so that's why the collectTonesResult replace the names.  The original DtmfTone is not existing at generated implementation model anymore